src/sentry/static/sentry/app/components/charts/miniBarChart.tsx

```diff
@@ -21,6 +21,18 @@ const defaultProps = {
    * Colors to use on the chart.
    */
   colors: [theme.gray400, theme.purple400] as string[],
+  /**
+   * Show max/min values on yAxis
+   */
+  labelYAxisExtents: false as boolean,
```

Comment:
is the type not inferred here?

Reply:
It can be inferred, I'll take the cast off.